app/services/idv/profile_maker.rb

```diff
@@ -18,10 +18,11 @@ def save_profile(
       profile.initiating_service_provider = initiating_service_provider
       profile.encrypt_pii(pii_attributes, user_password)
       profile.proofing_components = current_proofing_components
+      profile.fraud_pending_reason = fraud_pending_reason
       profile.save!
       profile.deactivate_for_gpo_verification if gpo_verification_needed
-      if fraud_pending_reason.present?
-        profile.deactivate_for_fraud_review(fraud_pending_reason: fraud_pending_reason)
+      if fraud_pending_reason.present? && !gpo_verification_needed
```

Comment:
I did some tracing through the code and couldn't find where `deactivate_for_fraud_review` will be called once the user enters their gpo code.

Reply:
That occurs in [`GpoVerifyForm#submit`](https://github.com/18F/identity-idp/pull/8821/files#diff-785c4e5b5e762463826af58d1461959c1119da2bde96260d394279f9c686a364R27).

Reply:
Ah, it's in a separate PR, that's why I couldn't find it on main! Thanks.